slices/python3-minimal.yaml

```diff
@@ -0,0 +1,29 @@
+package: python3-minimal
+
+essential:
+  - python3-minimal_copyright
+
+slices:
+  bins:
+    essential:
+      - python3-minimal_config
+      - python3-minimal_modules
+      - python3.12-minimal_bins
+    contents:
+      /usr/bin/py3clean:
+      /usr/bin/py3compile:
+      /usr/bin/py3versions:
+      /usr/bin/python3:
+
+  config:
+    contents:
+      /usr/share/python3/debian_defaults:
+
+  modules:
```

Comment:
why are these called "modules"?

Does `py3versions.py` need `debpython`? The former is related to the bin py3versions symlink, so they could stay together.

Reply:
debian_defaults needed by py3versions. The others are needed by py3compile/py3clean. Added more slices for those.